### DISCUSSION TITLE
Diamond database format

### DISCUSSION THREAD

**Discussion by blaizereal** [bbuchfink/diamond#574] [General]
Is Diamond's database created with makedb a custom format or does it follow some standard (e.g. SQL Lite) ? 

Thanks: B

**Comment by bbuchfink** [comment]
It's a custom binary format.

**Reply by blaizereal** [reply]
Thanks!  What's new is that the latest Linux 5.18RC1 kernel update and enabling software NUMA (with 8 nodes, AUTO selecting for sockets) balancing has not improved the multithreading performance.
